Neos.Neos/Classes/Controller/Frontend/NodeController.php

```diff
@@ -120,8 +125,14 @@ class NodeController extends ActionController
      * with unsafe requests from widgets or plugins that are rendered on the node
      * - For those the CSRF token is validated on the sub-request, so it is safe to be skipped here
      */
-    public function previewAction(string $node): void
+    public function previewAction(string $node, string $renderingModeName = null): void
```

Comment:
FYI, the ui will currently not use this parameter yet (it could via https://github.com/neos/neos-ui/pull/3411)

But importantly this change allows us to open a page in a new tab independent of the session's value.
Although this would only work for this one page, successive links will currently not have that query param added - the linking service still needs to lear this, but this is out of scope of this pr.